templates/catalog/category_products.html

```diff
@@ -2,6 +2,12 @@
 {% load se_extras %}
 {% load user_agents %}
 
+{% comment %} @todo #302:60m Implement markup of pagination buttons for seo purposes.
```

Comment:
currently, we don't have pagination buttons themselves.
So, it's not just about markup. Moreover, buttons will be hidden, so they require no markup.

This description might be good:
```
Implement pagination buttons for seo purposes.
 Create buttons with django templates on backend and hide them with js on frontend.
 Load_more button should reniew canonical url for current page.
 See the parent task for details.
```